examples/minimal-tanstack-start/package.json

```diff
@@ -8,9 +8,10 @@
     "start": "vinxi start"
   },
   "dependencies": {
-    "@tanstack/react-router": "^1.58.7",
-    "@tanstack/start": "^1.58.7",
-    "@uploadthing/react": "7.0.3",
+    "@tanstack/react-router": "^1.69.1",
+    "@tanstack/start": "^1.69.3",
+    "@uploadthing/react": "^7.0.3",
+    "@vitejs/plugin-react": "^4.3.2",
```

Comment:
why do this need to be a prod dep?

Reply:
Not sure, but for some reason it was changed in Tanstack's getting started: https://tanstack.com/router/latest/docs/framework/react/start/getting-started#install-dependencies